### PR TITLE
Preguntas Grupo 7

### PR DESCRIPTION
**Pull request by jcv-dev**
Se agregan las preguntas corerspondientes a los integrantes del grupo 7.

### CODE REVIEW

File: public/data/5555555-7777/info.json [hunk 1]
```diff
@@ -41,6 +41,4 @@
       ]
     }
   ]
-}
-
-  
\ No newline at end of file
+}
```
Comment (fredyballest) [inline public/data/5555555-7777/info.json]: Changes not required. Please exclude this file from the PR

**Comment by fredyballest** [comment]
Please exclude [public/data/5555555-7777/info.json] from PR

**Review by fredyballest — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by fredyunivalle** [comment]
Hi @jcv-dev  Please resolve the conflicts and resubmit. I'm sorry, but another group submitted ahead of you.

**Comment by jcv-dev** [comment]
@fredyballest I have resolved the conflict and added a new commit to the PR. Thank you for letting me know.

**Review by fredyunivalle — APPROVED** [review]
LGTM

**Review by fredyballest — APPROVED** [review]

**Comment by fredyballest** [comment]
@jcv-dev  Test has failed https://github.com/fredyunivalle/michinglishapp/actions/runs/11583524788/job/32249063352 please check it

**Comment by jcv-dev** [comment]
@fredyballest I have changed the question causing the problem. All tests should pass now. Sorry.

**Pull request merged**